archetypes/archetypes/src/main/archetype/mp/custom/database-outputs.xml

```diff
@@ -318,7 +318,7 @@ docker run --rm --name mysql -p 3306:3306 \
                     <value key="jdbcDataSource">com.mysql.cj.jdbc.MysqlDataSource</value>
                     <value key="databaseUrl">jdbc:mysql://127.0.0.1:3306/pokemon?useSSL=false</value>
                     <value key="dbUser">user</value>
-                    <value key="userPassword">password</value>
+                    <value key="userPassword">changeit</value>
```

Comment:
See the comment about `{{userPassword}}` above.

Reply:
regarding the previous case I agree - fixed. Here, hovewer, the variable name is ``userPassword`` and the value is ``password`` which was changed to ``changeit``. So, should be correct.